build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id("com.hivemq.extension")
+    id("org.asciidoctor.jvm.convert")
+}
+
+group = "com.hivemq"
+description = "Prometheus Monitoring Extension"
+
+hivemqExtension {
+    name = "Prometheus Monitoring Extension"
+    author = "HiveMQ"
+    priority = 0
+    startPriority = 0
+    sdkVersion = "${property("hivemq-extension-sdk.version")}"
+}
+
+tasks.hivemqExtensionResources {
+    from("LICENSE")
+    from("README.adoc") { rename { "README.txt" } }
+    from(tasks.asciidoctor)
+    from("src/main/resources/prometheusConfiguration.properties")
```

Comment:
can you instead move the properties file to the src/hivemq-extension/ folder and remove this line

Reply:
done

Reply:
This is still here, it should be removed